pkg/machinery/kernel/kernel.go

```diff
@@ -34,5 +41,12 @@ type Param struct {
 
 // Path returns the path to the systctl file under /proc/sys.
 func (prop *Param) Path() string {
```

Comment:
Since this function is now only applicable to sysctls and not sysfs's, should its name be changed to better reflect that limitation, now that this package supports both?

Reply:
At least, the following `/` prefix check relegates it to only safely operating with sysctls, it seems.

Reply:
I'm not sure I understand what is meant by the "old behavior" here, though... it just looks dangerous.

Reply:
I think I can rewrite that a bit to look more straightforward, but will need to update all of our sysctls definitions.

Reply:
should be good now.